### DISCUSSION TITLE
difference between "TF" regulon and "TF_extended" regulon?

### DISCUSSION THREAD

**Discussion by WeiCSong** [aertslab/SCENIC#256] [* General SCENIC questions] [answered]
Hi everyone, i'm currently using SCENIC step2&3 for my research, and i'm puzzled about "extended" regulon. I guess that one regulon is a subset of its "_extended" regulon, and has a higher threshold of determination of  regulation. Shall i choose the "_extended" regulon to study one TF? How shall i explain to the reviewer why i choose it?   I don't quite understand the biological meanings of their difference. Wish to learn from you. Thanks!

**Comment by mellyS** [comment]
I have the same question . Could you please explain us the difference between the two outputs ? Thanks !

**Comment by s-aibar** [comment, marked as answer]
Dear @WeiCSong  and @mellyS ,

The "extended" regulons include motifs that have been linked to the TF by lower confidence annotations (e.g. inferred by motif similarity). 

You can find the full explanation in the detailed vignette *[detailedStep_2_createRegulons.Rmd](https://rawcdn.githack.com/aertslab/SCENIC/a0a00644b2f3589a3e2bc65486fc5f6cc00f48e1/inst/doc/detailedStep_2_createRegulons.html)* : 

> 1.2 Annotate motifs to TFs
...
The annotations provided by the cisTarget databases can be divided into high-confidence or low-confidence, depending on the annotation source (annotated in the original database, inferred by orthology, or inferred by motif similarity). The main regulons only use the “high confidence” annotations, which by default are “direct annotation” and “inferred by orthology”. **The sufix _extended** in the regulon name indicates **lower confidence annotations** (by default “inferred by motif similarity”) are also used.

Sorry that is is not more clear in the main vignette. To avoid overcrowding it, I have just created a FAQ that includes this question.

Thank you for your feedback!
Sara

**Reply by yhdu36** [reply]
Hi, May I ask if is it possible to remove the TF_extended in the results? or is it not advised to do so?
